WordPress/Classes/ViewRelated/Blog/EditSiteViewController.m

```diff
@@ -1,3 +1,17 @@
+// Edit Site contents:
```

Comment:
Let's try to minimize stand alone comments such as this one.  If you need to document a particular element in the file you can use the doxygen documentation style that we're currently using in other sections of the app.

This is anyways documented by the definition of your `EditSiteRow` and `TableSectionContentType` enums, so I'd just remove these.